### ISSUE TITLE
[Accessibility]: Dark theme quote blocks fail Lighthouse contrast

### ISSUE THREAD

**Issue by AshleyHenry15** [posit-dev/product-doc-theme#119]
**Describe the accessibility issue**
A clear and concise description of what the accessibility is.

Blockquotes don't pass Lighthouse contrast for accessibility.

**Resolution**
If applicable, add the solution to resolve this accessibility issue.

**Additional context**
